build/media-suite_deps.sh

```diff
@@ -0,0 +1,107 @@
+SOURCE_REPO_LIBRIST=https://code.videolan.org/rist/librist.git
+SOURCE_REPO_LIBRTMP=https://gitlab.com/m-ab-s/rtmpdump.git
+SOURCE_REPO_LIBSOXR=https://gitlab.com/m-ab-s/libsoxr.git
+SOURCE_REPO_LIBTIFF=https://gitlab.com/libtiff/libtiff.git
+SOURCE_REPO_LIBVMAF=https://github.com/Netflix/vmaf.git
+SOURCE_REPO_LIBVORBIS=https://github.com/xiph/vorbis-tools.git
+SOURCE_REPO_LIBVVDEC=https://github.com/fraunhoferhhi/vvdec.git
+SOURCE_REPO_LIBVVENC=https://github.com/fraunhoferhhi/vvenc.git
+SOURCE_REPO_LIBWEBP=https://chromium.googlesource.com/webm/libwebp
+SOURCE_REPO_LIBXML2=https://gitlab.gnome.org/GNOME/libxml2.git
+SOURCE_REPO_LIBZEN=https://github.com/MediaArea/ZenLib.git
+SOURCE_REPO_LUAJIT=https://github.com/LuaJIT/LuaJIT.git
+SOURCE_REPO_MEDIAINFO=https://github.com/MediaArea/MediaInfo.git
+SOURCE_REPO_MPV=https://github.com/mpv-player/mpv.git
+SOURCE_REPO_MRUBY=https://github.com/mruby/mruby.git
+SOURCE_REPO_MUJS=https://github.com/ccxvii/mujs.git
+SOURCE_REPO_NEON=https://github.com/notroj/neon.git
+SOURCE_REPO_OPENAL=https://github.com/kcat/openal-soft.git
+SOURCE_REPO_OPENCLHEADERS=https://github.com/KhronosGroup/OpenCL-Headers.git
+SOURCE_REPO_OPUS=https://github.com/xiph/opus.git
+SOURCE_REPO_OPUSEXE=https://github.com/xiph/opus-tools.git
+SOURCE_REPO_OPUSFILE=https://github.com/xiph/opusfile.git
+SOURCE_REPO_RUBBERBAND=https://github.com/m-ab-s/rubberband.git
+SOURCE_REPO_SDL2=https://github.com/libsdl-org/SDL.git#SDL2
+SOURCE_REPO_SHADERC=https://github.com/google/shaderc.git
+SOURCE_REPO_SNDFILE=https://github.com/libsndfile/libsndfile.git
+SOURCE_REPO_SPEEX=https://github.com/xiph/speex.git
+SOURCE_REPO_SPIRV_CROSS=https://github.com/KhronosGroup/SPIRV-Cross.git
+SOURCE_REPO_SPIRV_HEADERS=https://github.com/KhronosGroup/SPIRV-Headers.git
+SOURCE_REPO_SPIRV_TOOLS=https://github.com/KhronosGroup/SPIRV-Tools.git
+SOURCE_REPO_SRT=https://github.com/Haivision/srt.git
+SOURCE_REPO_SVTAV1=https://gitlab.com/AOMediaCodec/SVT-AV1.git
+SOURCE_REPO_SVTHEVC=https://github.com/OpenVisualCloud/SVT-HEVC.git
+SOURCE_REPO_SVTVP9=https://github.com/OpenVisualCloud/SVT-VP9.git
+SOURCE_REPO_TESSERACT=https://github.com/tesseract-ocr/tesseract.git
+SOURCE_REPO_UAVS3D=https://github.com/uavs3/uavs3d.git
+SOURCE_REPO_UVG266=https://github.com/ultravideo/uvg266.git
+SOURCE_REPO_VIDSTAB=https://github.com/georgmartius/vid.stab.git
+SOURCE_REPO_VPX=https://chromium.googlesource.com/webm/libvpx
+SOURCE_REPO_VULKANLOADER=https://github.com/KhronosGroup/Vulkan-Loader.git
+SOURCE_REPO_VVC=https://gitlab.com/m-ab-s/VVCSoftware_VTM.git
+SOURCE_REPO_X264=https://code.videolan.org/videolan/x264.git
+SOURCE_REPO_X265=https://bitbucket.org/multicoreware/x265_git.git
+SOURCE_REPO_XAVS2=https://github.com/pkuvcl/xavs2.git
+SOURCE_REPO_XAVS=https://github.com/Distrotech/xavs.git
+SOURCE_REPO_XVC=https://github.com/divideon/xvc.git
+SOURCE_REPO_XVID=https://github.com/m-ab-s/xvid.git
+SOURCE_REPO_ZIMG=https://github.com/sekrit-twc/zimg.git
```

Comment:
newline missing

Reply:
editorconfig or clang-format mey help in future.

Reply:
> newline missing

I could have sworn that I already noticed and pushed an update to fix it. Must have dreamt it..
Fixed.

Reply:
> **editorconfig** or clang-format mey help in future.

Do you have one I could use?

Reply:
> > newline missing
> 
> I could have sworn that I already noticed and pushed an update to fix it. Must have dreamt it.. Fixed.

Ah - mystery solved. I actually did fix it (https://github.com/m-ab-s/media-autobuild_suite/compare/96aaf5ff83781b9c85e662a842c0e5dde5fe5dae..6d1dd135fdda14f6304543ede6a0e9390ee2066b) but then it got lost again when I had done the sorting.